src/components/timeline/index.js

```diff
@@ -7,10 +7,10 @@
 import * as React from 'react';
 import explicitConnect from 'firefox-profiler/utils/connect';
 import { getTimelineTrackOrganization } from 'firefox-profiler/selectors';
-import FullTimeline from 'firefox-profiler/components/timeline/FullTimeline';
-import ActiveTabTimeline from 'firefox-profiler/components/timeline/ActiveTabTimeline';
-import OriginsTimelineView from 'firefox-profiler/components/timeline/OriginsTimeline';
-import { assertExhaustiveCheck } from 'firefox-profiler/utils/flow';
+import { FullTimeline } from '../timeline/FullTimeline';
+import { ActiveTabTimeline } from '../timeline/ActiveTabTimeline';
+import { OriginsTimelineView } from '../timeline/OriginsTimeline';
```

Comment:
For these ones, you can replace `../timeline` with `./` simply, because they are from the same directory.